static/js/protein/page.tsx

```diff
@@ -100,22 +100,62 @@ export class Page extends React.Component<PagePropType, PageStateType> {
   }
 
   render(): JSX.Element {
+    /* this.props.nodeName is formatted as ProteinName_SpeciesName
+    Using the split we get the ProteinName and SpeciesName separately
+    */
+    const result = this.props.nodeName.split("_");
```

Comment:
nit: the variable name `result` isn't great here because there's no context for what result it's talking about. Maybe splitNodeName or processedNodeName or something about the name